admin-dev/themes/new-theme/package.json

```diff
@@ -18,37 +18,46 @@
     "bootstrap-tokenfield": "^0.12.0",
     "bourbon": "^4.2.6",
     "compass": "^0.1.1",
+    "copy-webpack-plugin": "^5.1.1",
```

Comment:
same for this one, in devDependencies (and below too)

Reply:
Same as my upper comment